lib/smart_answer_flows/student-finance-calculator/outcomes/outcome_uk_all_students.govspeak.erb

```diff
@@ -7,7 +7,7 @@
     To qualify for a Maintenance Loan you must be studying an Initial Teacher Training course (if it's degree level or above).
   <% elsif course_studied == 'none-of-the-above' %>
     To qualify for a Maintenance Loan, you must be studying one of the following courses:
-    
+
```

Comment:
is this line intentional? :)

Reply:
My editor deleting the trailing whitespace for me! I tend to leave them in.